frontend/src/metabase/nav/components/ProfileLink.jsx

```diff
@@ -85,7 +83,7 @@ function ProfileLink({ user, handleCloseNavbar, adminItems, handleLogout }) {
       {
         title: t`Sign out`,
         icon: null,
-        action: () => handleLogout(),
+        action: () => onLogout(),
```

Comment:
nit: What about passing `onLogout` directly?

```suggestion
        action: onLogout,
```

I checked that the callback doesn't expect any argument, so this should be fine.

https://github.com/metabase/metabase/blob/97026720e5b3fba31843af0125646a0a293616e3/frontend/src/metabase/auth/actions.ts#L67-L77